src/PasswordChecker.php

```diff
@@ -32,6 +33,11 @@ public function setMinLength(int $minLength): void
         }
     }
 
+    public function enableComplexityRequirements(): void
```

Comment:
It might be nice to give more control, rather just all or nothing.
`setComplexityRequirements(['lowercase', 'uppercase', 'number', 'symbol'])`